Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/LocalData.kt

```diff
@@ -357,6 +357,34 @@ object LocalData {
             )
         }
 
+    /**
+     * Gets a boolean that tells weather the risk level
```

Comment:
"whether", not :partly_sunny:

Reply:
I always mix these two up.  It's fixed now